static/theme-assets/editor/job-preview.js

```diff
@@ -22,17 +22,33 @@ const renderBisList = function (bis) {
     "div", {}, bisList.map(function (bis, indexer) {
         const name = h("h2", {}, bis.name);
         const type = bis.type;
-        const link = bis.link;
         const description = h("p", {}, bis.description);
-        const bisFrame =
-          type != "plaintext"
-            ? h("div", { class: "h-96" }, h("iframe", {
-                src: 
-                  type === "etro" ? `https://etro.gg/embed/gearset/${link}` : 
-                  link,
-                class: "w-full h-full",
-              }))
-              : link;
+        let link = bis.link;
+        
+        switch(type) {  
+          case "etro":  
+            const etroLink = link.match(/etro.gg\/gearset\/([A-Za-z0-9-]+)(?:[?#]|$)/i);  
+            if(!etroLink) {  
+              link = `https://etro.gg/embed/gearset/${link}`  
+            }
```

Comment:
```suggestion
            if(etroLink) {
              link = etroLink[1]
            }
            // Make etro links always become the embedded form
            link = `https://etro.gg/embed/gearset/${link}`
```

Sorry, misread this behaviour slightly in the last suggestion.